src/reports/package/root/README.md

```diff
@@ -0,0 +1,64 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+# accessibility-insights-report
+
+Publish [axe-core](https://github.com/dequelabs/axe-core) accessibility scan results as 
+self-contained HTML files in the same format as
+[accessibility-insights-web](https://github.com/microsoft/accessibility-insights-web).
+
+## Usage
+
+Before using accessibility-insights-report, you will need to run an [axe](https://github.com/dequelabs/axe-core) accessibility scan to produce some axe results to convert. Typically, you would do this by using an axe integration library for your favorite browser automation tool ([axe-puppeteer](https://github.com/dequelabs/axe-puppeteer), [axe-webdriverjs](https://github.com/dequelabs/axe-webdriverjs), [cypress-axe](https://github.com/avanslaars/cypress-axe)).
+
+axe-sarif-converter exports a factory that can be used to create a report object and output its results as HTML.
+
+Use it like this:
+
+```ts
+import * as Axe from 'axe';
+import * as AxePuppeteer from 'axe-puppeteer';
+import * as fs from 'fs';
+import * as Puppeteer from 'puppeteer'
+import * as util from 'util';
+
+import { reporterFactory } from "accessibility-insights-report";
+
+test('my accessibility test', async () => {
+    // This example uses axe-puppeteer, but you can use any axe-based library
+    // that outputs axe scan results in the default axe output format
+    const testPage: Puppeteer.Page = /* ... set up your test page ... */;
+    const axeResults: Axe.AxeResults = await new AxePuppeteer(testPage).analyze();
+
+    // Perform the conversion
+    const html = reporter.fromAxeResult(axeResults).asHTML();
```

Comment:
should this be reporterFactory ?

Reply:
Actually there was a missing line. Good catch. Thanks.

Fixed.